lib/std/http/Client.zig

```diff
@@ -1579,7 +1584,7 @@ pub fn fetch(client: *Client, allocator: Allocator, options: FetchOptions) !Fetc
     return res;
 }
 
-test {
+test "parseInt3" {
```

Comment:
this is supposed to be `test {` so that the references in this block apply even when --test-filter is used.

Reply:
Correct, I needed to change this during testing and forgot to revert.